### PR TITLE
add reload method completes #102 

### PR DESCRIPTION
**Pull request by seancookr**
#102 

extract a common method needed in many applications to the base document

### CODE REVIEW

File: test/dummy/log/test.log [hunk 1]
```diff
@@ -0,0 +1,1470 @@
+--------------------------------------------------
```
Comment (javierg) [inline test/dummy/log/test.log]: left over file?

**Comment by netlogics** [comment]
Looks good. :shipit: unless @javierg has any comments.

File: lib/dolly/document.rb [hunk 2]
```diff
@@ -36,6 +36,10 @@ def update_properties! options = {}
       save
     end
 
+    def reload
+      self.doc = self.class.find(id).doc
```
Comment (javierg) [inline lib/dolly/document.rb]: I dont think you need the self in doc... also, please add to the test the check that accessors are updated

**Comment by javierg** [comment]
@seancookr :-( did you confirm that accessors were updated?

**Pull request merged**